engine/jobmaster/dm/task_manager.go

```diff
@@ -51,19 +52,27 @@ type TaskManager struct {
 }
 
 // NewTaskManager creates a new TaskManager instance
-func NewTaskManager(initTaskStatus []runtime.TaskStatus, jobStore *metadata.JobStore, messageAgent dmpkg.MessageAgent, pLogger *zap.Logger, metricFactory promutil.Factory) *TaskManager {
+func NewTaskManager(
+	jobID string,
+	initTaskStatus []runtime.TaskStatus,
+	jobStore *metadata.JobStore,
+	messageAgent dmpkg.MessageAgent,
+	pLogger *zap.Logger,
+	metricFactory promutil.Factory,
+) *TaskManager {
 	taskManager := &TaskManager{
+		jobID:         jobID,
 		DefaultTicker: ticker.NewDefaultTicker(taskNormalInterval, taskErrorInterval),
 		jobStore:      jobStore,
 		logger:        pLogger.With(zap.String("component", "task_manager")),
 		messageAgent:  messageAgent,
 		gaugeVec: metricFactory.NewGaugeVec(
```

Comment:
@lance6716 @okJiang Please review this change.

Reply:
I think it's OK

Reply:
ok for me